apps/server/src/modules/h5p-library-management/service/h5p-library-management.service.spec.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Test, TestingModule } from '@nestjs/testing';
+import { DeepMocked, createMock } from '@golevelup/ts-jest';
+import { ContentStorage, LibraryStorage } from '@src/modules/h5p-editor/service';
+import { LibraryAdministration, ContentTypeCache } from '@lumieducation/h5p-server';
+import { IHubContentType } from '@lumieducation/h5p-server/build/src/types';
+import { s3ConfigContent, s3ConfigLibraries } from '@src/modules/h5p-editor/h5p-editor.config';
+import { H5PLibraryManagementService } from './h5p-library-management.service';
+
+jest.mock('@lumieducation/h5p-server', () => {
+	return {
+		...jest.requireActual('@lumieducation/h5p-server'),
+		LibraryAdministration: jest.fn().mockImplementation(() => {
+			return {
+				getLibraries: jest.fn().mockResolvedValue([
+					{ machineName: 'a', dependentsCount: 0 },
+					{ machineName: 'b', dependentsCount: 1 },
+					{ machineName: 'c', dependentsCount: 0 },
+				]),
+			};
+		}),
+	};
+});
+
+describe('H5PLibraryManagementService', () => {
```

Comment:
For the thr-18 PR we are forced to merge it without checking the test files. 
But for new written code. Please check our guidlines and examples in code. We use explizit setup methodes that bind directly to a when describe with a senario description. 

In the setup method are all mocks and data preparations are placed.

Reply:
solved: c6e04bc09de60c0a3e60dfd9559f2eb37e558a80

Reply:
Let speak together, or check the confluence document "Code Styles 08.02.23" or the moved documentation https://github.com/hpi-schul-cloud/schulcloud-documentation/pull/16/files#diff-b0fa6054ace0d1d213f15f9215e145ac1c2e288d54f94d077dca3d6141125125R75

Reply:
Solved: 98c0bf2b3264ae904cbf8c37821d3e7881976db1